### ISSUE TITLE
[Bug]Slimtoolkit doesn't recognised docker engine comes buddled up with Docker Desktop

### ISSUE THREAD

**Issue by StarTrooper08** [slimtoolkit/slim#519]
Expected Behavior
=================
The slimtoolkit should connect with docker desktop and docker engine but it didn't!

---
Actual Behavior
=================

I've installed Docker Desktop which comes bundled up with Docker Engine and then installed slimtoolkit but I think slimtoolkit didn't able to find docker engine which comes with Docker desktop.
So It only connects with docker when we install docker engine and other docker peripherals when we install it seprately.

---
Steps to Reproduce the Problem
===============================
 

  1.
  1.
  1.

---
Specifications
=================
  - Version: Slimtoolkit version 1.40.1
  - Platform: Ubuntu Linux 22.04 LTS

**Comment by kcq** [comment]
Thanks for sharing your problem @StarTrooper08 ! This might be linux desktop related. Not a problem with the Mac Docker Desktop. `slim` does need `/var/run/docker.sock`, which is the traditional docker socket path, but it's possible that it's not created in your Docker Desktop installation. According to https://github.com/docker/desktop-linux/issues/20 there's a linux Docker Desktop option for it: `Settings -> Advanced -> check the option Enable default Docker socket (Requires password)`. Let me know if this does the trick for you.

**Comment by kcq** [comment]
@StarTrooper08 adding the Docker Desktop notes to the README

**Comment by kcq** [comment]
here's the info: https://github.com/slimtoolkit/slim#docker-desktop

**Comment by kcq** [comment]
There's a small enhancement that can go along with the new Docker Desktop readme notes (to try `~/.docker/run/docker.sock` if `/var/run/docker.sock` is not there. Once it's added this ticket can be closed.

**Comment by kcq** [comment]
the enhancement will be available in the next release (already in master)

**Comment by kcq** [comment]
@DenisBY yes, that's one way to do it by using a custom `--host` flag to point to the OrbStack's Docker socket. Another option to consider is the `--crt-context`, which is like Docker's `--context` flag where it can look like this: `--crt-context=orbstack`

**Comment by kcq** [comment]
This issue is pretty much resolved and the `--host` flag is a good backup along with the `--crt-context` flag if the container runtime you use configures Docker context (like `OrbStack` does. `LIMA` and `COLIMA` do this too).
